include/seastar/http/routes.hh

```diff
@@ -120,6 +120,14 @@ public:
      */
     routes& add(operation_type type, const url& url, handler_base* handler);
 
+    /**
+     * Add a global handler:
+     * Example  routes.add_global_hanlder(handler);
+     * @param handler
+     * @return
+     */
+    routes& add_global_handler(handler_base* handler);
```

Comment:
Would be good to clarify what a "global" handler mean. I guess that this handler will get any HTTP method (unfortunately a new name "operation_type" was invented for that in the code) and URL? Maybe the word default, instead of global, is better, if you allow specific routes to be overridden in addition to that default (but currently you don't - more on this below).

Reply:
We do support general path definitions (it's used for serving files for example) my question is there a need for a general operation?

Reply:
Subject line and comments for the PR updated with explanation and code updated based on the review comments